examples/cluster/plot_dbscan.py

```diff
@@ -1,39 +1,53 @@
-# -*- coding: utf-8 -*-
 """
 ===================================
 Demo of DBSCAN clustering algorithm
 ===================================
 
-DBSCAN (Density-Based Spatial Clustering of Applications with Noise)
-finds core samples of high density and expands clusters from them.
-This algorithm is good for data which contains clusters of similar density.
+DBSCAN (Density-Based Spatial Clustering of Applications with Noise) finds core
+samples of high density and expands clusters from them. This algorithm is good
```

Comment:
What is "core samples of high density"? The thing that throws me off is that a sample can't have a density, because a density is something like "number of samples / unit volume".

I'm not familiar enough with the algorithm to suggest a fix. It sounds like maybe it should be something like "core samples in regions of high density"?

Wikipedia starts explaining DBSCAN as: "It is a [density-based clustering](https://en.wikipedia.org/wiki/Cluster_analysis#Density-based_clustering) non-parametric algorithm: given a set of points in some space, it groups together points that are closely packed together (points with many [nearby neighbors](https://en.wikipedia.org/wiki/Fixed-radius_near_neighbors)),"

Reply:
You are right, the current wording is not exact. Thanks for the catch!